examples/charts/brushing_zooming.htm

```diff
@@ -0,0 +1,166 @@
+            <div class='row trunk-section'>
+                <div class='col-lg-7'>
+                    <div class='row'>
+                        <div class='col-lg-12 text-center' id='basic_brushing'></div>
+                        <div class='col-lg-12 text-center' id='point_chart_brushing'></div>
+                    </div>
+                </div>
+                <div class='col-lg-5'>
+                    <div class='data-column'><a href='data/fake_users3.json'>data</a></div>
+
+<pre><code class='javascript'>d3.json('data/fake_users2.json', function(data) {
+    for (var i = 0; i < data.length; i++) {
+        data[i] = MG.convert.date(data[i], 'date');
+    }
+    MG.data_graphic({
+        title: "Basic Brushing & Zooming",
+        description: "This is a simple for brushing and zooming feature. You can set 'brush' as 'xy', 'x', 'y' to restrict axis.",
```

Comment:
"This is a simple for brushing and zooming feature. You can set 'brush' as 'xy', 'x', 'y' to restrict axis." => "This is a simple example of brushing and zooming. You can set 'brush' as 'xy', 'x', 'y' to specify the axis(es) that may be brushed."